airbyte-integrations/connectors/destination-s3/src/main/kotlin/io/airbyte/integrations/destination/s3/S3DestinationAsync.kt

```diff
@@ -9,7 +9,7 @@ import io.airbyte.cdk.integrations.destination.s3.BaseS3Destination
 import io.airbyte.cdk.integrations.destination.s3.S3DestinationConfigFactory
 import io.airbyte.cdk.integrations.destination.s3.StorageProvider
 
-open class S3Destination : BaseS3Destination {
```

Comment:
I don't understand the renaming. The name `S3DestinationAsync` suggest there's a non-async one that's in the same repo, but once this is merged, there's still only one `S3Destination` that happens to be "async" (which doesn't mean much in the context of a destination)

Reply:
Yeah, I renamed it and let IntelliJ propagate the name so I could see what I was touching. But you're right that it isn't strictly necessary.